style.css

```diff
@@ -0,0 +1,10 @@
+.{
+    margin: 0px;
+    padding: 0px;
+    
+}
+.header{
+    height: 100px;
+    width: 200px;
+    
+}
```

Comment:
style errors, check the Linters error listed above:

style.css
  1:1   ✖  Expected single space before "{"             block-opening-brace-space-before     
  2:5   ✖  Expected indentation of 2 spaces             indentation                          
  2:14  ✖  Unexpected unit                              length-zero-no-unit                  
  3:5   ✖  Expected indentation of 2 spaces             indentation                          
  3:15  ✖  Unexpected unit                              length-zero-no-unit                  
  4:4   ✖  Unexpected whitespace at end of line         no-eol-whitespace                    
  5:1   ✖  Unexpected empty line before closing brace   block-closing-brace-empty-line-before
  6:1   ✖  Expected empty line before rule              rule-empty-line-before               
  6:7   ✖  Expected single space before "{"             block-opening-brace-space-before     
  7:5   ✖  Expected indentation of 2 spaces             indentation                          
  8:5   ✖  Expected indentation of 2 spaces             indentation                          
  9:4   ✖  Unexpected whitespace at end of line         no-eol-whitespace                    
 10:1   ✖  Unexpected empty line before closing brace   block-closing-brace-empty-line-before
 10:1   ✖  Unexpected missing end-of-source newline     no-missing-end-of-source-newline